.eslintrc.js

```diff
@@ -15,6 +17,8 @@ module.exports = {
     '@typescript-eslint/no-namespace': 'off',
     '@typescript-eslint/explicit-member-accessibility': 'warn',
     '@typescript-eslint/interface-name-prefix': 'off',
+    '@typescript-eslint/ban-types': 'off',
```

Comment:
Do you have reasons for why we needed to turn off these two rules?

Reply:
It gives ban-type errors in communication.ts files when we build the repo. We have these rules disabled in communication.ts in metaos repo. I will delete these lines from here, and will put in the communication file, so we wont be banning them for all files. Is that good?